Nathan quality of life update

nnard1616 opened the pull request:
Some quality of life updates:

1.  Moved Chapters and appendices into their respective directories, this enables the use of a README.md for github that doesn't get printed into the pdf.
2.  Re organized the chapters so that Chapter 3 -> 3-person patterns, Chapter 4 -> 4-person patterns, etc.  I can't tell you how much this bothered me! XD
3.  Updated the tex template a little so that Appendices are more distinct from the chapters.  
4. Created a \majpatternbookversion latex command to print the current book version supplied by the Makefile.
5. Updated the README.md to have more useful information regarding building the pdf.
6.  Added little pieces of flair here and there

nnard1616 commented on 11_hexclover.md:
```diff
@@ -1,28 +0,0 @@
-## Hex of Clovers
```
This is a subsection in chapter 11, so for organizational purposes  I have moved this to the end of 11_big_patterns.md

nnard1616 commented on 17_piano_rolls.md:
```diff
@@ -1,5 +0,0 @@
-# Nathan's Piano Rolls
```
We both made this, I had appendix_a_piano_rolls.md previously.

nnard1616 commented on appendices/appendix_a_piano_rolls.md:
```diff
@@ -0,0 +1,3 @@
+# Piano Rolls
```
I am thinking appendix A should be a listing of site swaps for each pattern in the book.  Something that readers can input into the Juggling Lab easily.

I think this might be better as appendix h, I had made it appendix a so that Graham's would be appendix g .  B^D

nnard1616 commented on README.md:
```diff
@@ -0,0 +1,28 @@
+![](./media/Standard_4_Person_Clover_Animation.gif)
```
If you think this animated gif is too distracting, we can definitely remove it.   So no worries if you prefer to delete it.

nnard1616 commented on README.md:
```diff
@@ -0,0 +1,28 @@
+latest *official* version in PDF format from [our web
+site](http://madjugglers.com/Madison%20Pattern%20Book).
+
+Dependencies
```
Feel free to add other dependencies that I have missed.

nnard1616 commented on appendices/appendix_0.md:
```diff
@@ -0,0 +1,9 @@
+Read documentation on the appendix package for better understanding of above commands:
+https://ctan.org/pkg/appendix?lang=en
+-->
+Below follows several extras and goodies that were unable to find a home in previous sections.  Things like additional patterns, observations, and diagrahams. 
```
any misspellings are intentional. B^D

nnard1616 commented on appendices/appendix_0.md:
```diff
@@ -0,0 +1,9 @@
+\appendix
+\clearpage 
+\appendixpage
+\addappheadtotoc
```
Unfortunately I couldn't figure out how to get the appendices to nest together under "Appendix" in the bookmark tree.  Instead they print like chapters, but at least there's a "Appendix" marker separating the chapters from appendices.  We might be able to get latex to force these to nest with each other if we dig deep enough.

nnard1616 commented on appendices/appendix_f_passing_43.md:
```diff
@@ -126,23 +126,3 @@ RR3 requires RR4
 
 58 total pattern combinations
 
-## More Patterns by  Graham
```
moved to appendix g

nnard1616 commented on chapters/01_maj_pattern_book.md:
```diff
@@ -0,0 +1,62 @@
+
+
+All jugglers are welcome to play with, use, distribute, extend, and otherwise enjoy all 
+the club-passing patterns described in *Madison Area Jugglers' Pattern Book, Version \majpatternbookversion ,* which is licensed by its creators, the Madison Area Jugglers, under the [Creative Commons Attribution 3.0 Unported License](http://creativecommons.org/licenses/by/3.0/deed.en_US) ![Creative Commons License](http://i.creativecommons.org/l/by/3.0/88x31.png){width=7%}. 
```
purpose of \majpatternbookversion to help reduce "hunting down" outdated version numbers to update them.  It's a little finicky though, may not be worth it.  Totally fine with removing this if you find it unnecessary.

nnard1616 commented on chapters/02_hello.md:
```diff
@@ -1,65 +1,3 @@
-# About this document 
```
I split what was chapter 1 into two chapters, so that Chapter 3 -> 3-Person Patterns, etc.

nnard1616 commented on chapters/02_hello.md:
```diff
@@ -98,7 +36,7 @@ Melonhead began tracking the patterns we do at  Thursday and Sunday MAJ practice
 to figure out what we really did.  But most of us forget the code words after a while,
 except for the frequently juggled ones, and this was impetus for the  Version 3.0
-update by Mike Newton in 2018.  Since then, we've suffered through two years of COVID restrictions and two lost Madfests.   But we kept juggling and discovered some cool new patterns that work well with face masks!  Nathan Nard's rejuvenating and inventive contributions have helped to launch this recent Version 3.1.
+update by Mike Newton in 2018.  Since then, we've suffered through two years of COVID restrictions and two lost Madfests.   But we kept juggling and discovered some cool new patterns that work well with face masks!  Nathan Nard's rejuvenating and inventive contributions have helped to launch this recent Version \majpatternbookversion.
```
purpose of \majpatternbookversion to help reduce "hunting down" outdated version numbers to update them.  It's a little finicky though, may not be worth it.  Totally fine with removing this if you find it unnecessary.

nnard1616 commented on chapters/06_six_person_patterns.md:
```diff
@@ -1,4 +1,4 @@
-# 6 Person Patterns
+# 6-Person Patterns
```
!!!!!!!!!!!!!!!!!!!

wiscstatman commented:
my quality of life has improved!

wiscstatman commented:
successfully merged!

thanks a million

I've got to get back to regular work this week, but I'm sure I'll steal some time for further updates

and I expect to be juggling on Thursday (are we outside?)

-Mike


Professor and Chair of Biostatistics and Medical Informatics
Professor of Statistics
www.stat.wisc.edu/~newton/
________________________________
From: Nathan Nard ***@***.***>
Sent: Sunday, May 1, 2022 11:40 PM
To: wiscstatman/pattern-book ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [wiscstatman/pattern-book] Nathan quality of life update (PR #3)


Some quality of life updates:

  1.  Moved Chapters and appendices into their respective directories, this enables the use of a README.md for github that doesn't get printed into the pdf.
  2.  Re organized the chapters so that Chapter 3 -> 3-person patterns, Chapter 4 -> 4-person patterns, etc. I can't tell you how much this bothered me! XD
  3.  Updated the tex template a little so that Appendices are more distinct from the chapters.
  4.  Created a \majpatternbookversion latex command to print the current book version supplied by the Makefile.
  5.  Updated the README.md to have more useful information regarding building the pdf.
  6.  Added little pieces of flair here and there

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/wiscstatman/pattern-book/pull/3

Commit Summary

  *   99c39e2<https://github.com/wiscstatman/pattern-book/pull/3/commits/99c39e2df7f8bffd5774278036f5410d25dba69a> Organizational changes mostly affecting Appendix
  *   0ca90b9<https://github.com/wiscstatman/pattern-book/pull/3/commits/0ca90b9b84ac5a8af347492b9657567f8062486d> some afterthought changes
  *   645e518<https://github.com/wiscstatman/pattern-book/pull/3/commits/645e518c4205215819d7ecc4eb6239441517f848> created latex command to print book version number

File Changes

(26 files<https://github.com/wiscstatman/pattern-book/pull/3/files>)

  *   D 11_hexclover.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-b54ca6b095f1e26fca25c160aad4b45f691048b02dd9d00d0f4f25da15deec18> (28)
  *   D 17_piano_rolls.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-7a7f3049fb6af88c4835ca44ca504b371a26c2670d0459f7b717614283607628> (5)
  *   M Makefile<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52> (6)
  *   A README.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5> (28)
  *   A appendices/appendix_0.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-568008c4a31f81461f193c30d29aea053abe07e7ef178e0054b7c8a4a84c7271> (9)
  *   A appendices/appendix_a_piano_rolls.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-5868d7d6544f52c629212599f62bd3bceac10506edaad70698a8c13213479017> (3)
  *   R appendices/appendix_b_dual_passing.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-4fc6979e2aa0ba3f94ee4cfb955bd585d2beda5176bf2d64d6855c3e2b0e3933> (9)
  *   R appendices/appendix_c_gandini_weaves.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-a08cddaa580fe77def4a33ed2583f9783df52ae47bb20514a9cbb95c794a5277> (2)
  *   R appendices/appendix_d_7club2ct.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-6f77e48eb9cabad45b680d6a90e274ab5c6dc0295339217ea32cd47c9313e996> (5)
  *   R appendices/appendix_e_synchronous_passing.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-85e7d95841d2d1c7f99899bbeed09117c82eb9333cdcb16e21bff47fc5b0fd55> (2)
  *   R appendices/appendix_f_passing_43.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-674f9a98f6664310ca209f1618fc5975723aadf031649744bcbc74b37765fcbc> (22)
  *   A appendices/appendix_g_graham.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-dac9790545233367a8f642cf2c95067fc4037331f49c36f840848aaa2403f4fa> (20)
  *   M build/majbook.pdf<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-1398ec4d583171e067f1ec33cf9cb02722364c809b68f25a61f2b0763e8d1438> (0)
  *   A chapters/01_maj_pattern_book.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-759603f1eeee3456b6a41fc097b7067142430b781ca821073df41cf38404d521> (62)
  *   R chapters/02_hello.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-d190e9ea2a529a7a636c87d4931f70283587a9e43039d191cb44c1542e63b475> (64)
  *   R chapters/03_three_person_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-a05bc22f168ba92e2670cb77b48246a78338531e24064f4cf60035f3d43335ea> (0)
  *   R chapters/04_four_person_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-6e29d1ce5011b04166f5a09c03ed387a638c03573f9da47df34f546c581d3029> (0)
  *   R chapters/05_five_person_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-23cf328d96e874992a9c9dbc1ed0395507e6649bb2afb677544c3d030672468f> (0)
  *   R chapters/06_six_person_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-3e0c54db0dd51e65f488c4a525eabcb219e2ec91a1e16c432681e8cf5dc1b4b2> (2)
  *   R chapters/07_seven_person_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-b98293881bee85c0e464063c836fb864033531172eafe79ac3f44c091aea55d8> (2)
  *   R chapters/08_eight_person_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-3a5da7252d285b5356541d78504f9a19acc2bf5a7558bfe541483ab7c25afc34> (2)
  *   R chapters/09_nine_person_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-699e2a8a75a7433d3e63465ef913ef31e74539aeba1cdef21fe71b096d6a7258> (2)
  *   R chapters/10_ten_person_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-43c7af99bbda4a21c4cefd78750df9496032c5fd2352f5a29195ab95358806d2> (2)
  *   R chapters/11_big_patterns.md<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-2e08ab0481a42db1b5e54810e8615dfcede31d8f0490ae37c947262155a2b3f5> (29)
  *   A media/Standard_4_Person_Clover_Animation.gif<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-90173d3e3cd9f1d27d5ba181bc01a2c95bd668a15fbb3b06ee5791f55dbc2b4a> (0)
  *   M pdf/template.tex<https://github.com/wiscstatman/pattern-book/pull/3/files#diff-00152e3d1c3d42ceac0eee13cf1a699cfce8d924e46a80f56df260209ff5ffcf> (5)

Patch Links:

  *   https://github.com/wiscstatman/pattern-book/pull/3.patch
  *   https://github.com/wiscstatman/pattern-book/pull/3.diff

—
Reply to this email directly, view it on GitHub<https://github.com/wiscstatman/pattern-book/pull/3>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABQ24SL2MBDOEEK533KMXODVH5MFLANCNFSM5U2535DQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

nnard1616 commented:
Awesome!

Yes, we are outside this Thursday.  I think we'll be at the MCS on Sundays
from now on, but that may still be up for debate.

See ya Thursday.

On Mon, May 2, 2022, 08:02 Michael A. Newton ***@***.***>
wrote:

> successfully merged!
>
> thanks a million
>
> I've got to get back to regular work this week, but I'm sure I'll steal
> some time for further updates
>
> and I expect to be juggling on Thursday (are we outside?)
>
> -Mike
>
>
> Professor and Chair of Biostatistics and Medical Informatics
> Professor of Statistics
> www.stat.wisc.edu/~newton/
> ________________________________
> From: Nathan Nard ***@***.***>
> Sent: Sunday, May 1, 2022 11:40 PM
> To: wiscstatman/pattern-book ***@***.***>
> Cc: Subscribed ***@***.***>
> Subject: [wiscstatman/pattern-book] Nathan quality of life update (PR #3)
>
>
> Some quality of life updates:
>
> 1. Moved Chapters and appendices into their respective directories, this
> enables the use of a README.md for github that doesn't get printed into the
> pdf.
> 2. Re organized the chapters so that Chapter 3 -> 3-person patterns,
> Chapter 4 -> 4-person patterns, etc. I can't tell you how much this
> bothered me! XD
> 3. Updated the tex template a little so that Appendices are more distinct
> from the chapters.
> 4. Created a \majpatternbookversion latex command to print the current
> book version supplied by the Makefile.
> 5. Updated the README.md to have more useful information regarding
> building the pdf.
> 6. Added little pieces of flair here and there
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/wiscstatman/pattern-book/pull/3
>
> Commit Summary
>
> * 99c39e2<
> https://github.com/wiscstatman/pattern-book/pull/3/commits/99c39e2df7f8bffd5774278036f5410d25dba69a>
> Organizational changes mostly affecting Appendix
> * 0ca90b9<
> https://github.com/wiscstatman/pattern-book/pull/3/commits/0ca90b9b84ac5a8af347492b9657567f8062486d>
> some afterthought changes
> * 645e518<
> https://github.com/wiscstatman/pattern-book/pull/3/commits/645e518c4205215819d7ecc4eb6239441517f848>
> created latex command to print book version number
>
> File Changes
>
> (26 files<https://github.com/wiscstatman/pattern-book/pull/3/files>)
>
> * D 11_hexclover.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-b54ca6b095f1e26fca25c160aad4b45f691048b02dd9d00d0f4f25da15deec18>
> (28)
> * D 17_piano_rolls.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-7a7f3049fb6af88c4835ca44ca504b371a26c2670d0459f7b717614283607628>
> (5)
> * M Makefile<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52>
> (6)
> * A README.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5>
> (28)
> * A appendices/appendix_0.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-568008c4a31f81461f193c30d29aea053abe07e7ef178e0054b7c8a4a84c7271>
> (9)
> * A appendices/appendix_a_piano_rolls.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-5868d7d6544f52c629212599f62bd3bceac10506edaad70698a8c13213479017>
> (3)
> * R appendices/appendix_b_dual_passing.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-4fc6979e2aa0ba3f94ee4cfb955bd585d2beda5176bf2d64d6855c3e2b0e3933>
> (9)
> * R appendices/appendix_c_gandini_weaves.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-a08cddaa580fe77def4a33ed2583f9783df52ae47bb20514a9cbb95c794a5277>
> (2)
> * R appendices/appendix_d_7club2ct.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-6f77e48eb9cabad45b680d6a90e274ab5c6dc0295339217ea32cd47c9313e996>
> (5)
> * R appendices/appendix_e_synchronous_passing.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-85e7d95841d2d1c7f99899bbeed09117c82eb9333cdcb16e21bff47fc5b0fd55>
> (2)
> * R appendices/appendix_f_passing_43.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-674f9a98f6664310ca209f1618fc5975723aadf031649744bcbc74b37765fcbc>
> (22)
> * A appendices/appendix_g_graham.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-dac9790545233367a8f642cf2c95067fc4037331f49c36f840848aaa2403f4fa>
> (20)
> * M build/majbook.pdf<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-1398ec4d583171e067f1ec33cf9cb02722364c809b68f25a61f2b0763e8d1438>
> (0)
> * A chapters/01_maj_pattern_book.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-759603f1eeee3456b6a41fc097b7067142430b781ca821073df41cf38404d521>
> (62)
> * R chapters/02_hello.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-d190e9ea2a529a7a636c87d4931f70283587a9e43039d191cb44c1542e63b475>
> (64)
> * R chapters/03_three_person_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-a05bc22f168ba92e2670cb77b48246a78338531e24064f4cf60035f3d43335ea>
> (0)
> * R chapters/04_four_person_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-6e29d1ce5011b04166f5a09c03ed387a638c03573f9da47df34f546c581d3029>
> (0)
> * R chapters/05_five_person_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-23cf328d96e874992a9c9dbc1ed0395507e6649bb2afb677544c3d030672468f>
> (0)
> * R chapters/06_six_person_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-3e0c54db0dd51e65f488c4a525eabcb219e2ec91a1e16c432681e8cf5dc1b4b2>
> (2)
> * R chapters/07_seven_person_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-b98293881bee85c0e464063c836fb864033531172eafe79ac3f44c091aea55d8>
> (2)
> * R chapters/08_eight_person_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-3a5da7252d285b5356541d78504f9a19acc2bf5a7558bfe541483ab7c25afc34>
> (2)
> * R chapters/09_nine_person_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-699e2a8a75a7433d3e63465ef913ef31e74539aeba1cdef21fe71b096d6a7258>
> (2)
> * R chapters/10_ten_person_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-43c7af99bbda4a21c4cefd78750df9496032c5fd2352f5a29195ab95358806d2>
> (2)
> * R chapters/11_big_patterns.md<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-2e08ab0481a42db1b5e54810e8615dfcede31d8f0490ae37c947262155a2b3f5>
> (29)
> * A media/Standard_4_Person_Clover_Animation.gif<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-90173d3e3cd9f1d27d5ba181bc01a2c95bd668a15fbb3b06ee5791f55dbc2b4a>
> (0)
> * M pdf/template.tex<
> https://github.com/wiscstatman/pattern-book/pull/3/files#diff-00152e3d1c3d42ceac0eee13cf1a699cfce8d924e46a80f56df260209ff5ffcf>
> (5)
>
> Patch Links:
>
> * https://github.com/wiscstatman/pattern-book/pull/3.patch
> * https://github.com/wiscstatman/pattern-book/pull/3.diff
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/wiscstatman/pattern-book/pull/3>, or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/ABQ24SL2MBDOEEK533KMXODVH5MFLANCNFSM5U2535DQ
> >.
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wiscstatman/pattern-book/pull/3#issuecomment-1114821193>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABLHYHJ2YGRJ5B6OJ3AUXQDVH7G7BANCNFSM5U2535DQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
